tests/shell/tests.py

```diff
@@ -2,10 +2,24 @@
 import unittest
 
 from django import __version__
-from django.core.management import call_command
+from django.core.management import CommandError, call_command
 from django.test import SimpleTestCase, mock
 from django.test.utils import captured_stdin, captured_stdout, patch_logger
 
+try:
+    import IPython
+    del IPython
```

Comment:
I don't see why these `del`s are needed - just a vague worry about leaving them in the module namespace?